### PR TITLE
LibSass, thank god.

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
speed racer

### CODE REVIEW

File: scss/_utilities/_variables.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@ $desktop: $larger;
 $susy: (
   container: 1440px,
   columns: 16,
-  gutters: 24px/64.5px,
+  gutters: (24/64.5),
```
Comment (weerd) [inline scss/_utilities/_variables.scss]: it s a trap

Comment (DFurnes) [inline scss/_utilities/_variables.scss]: :facepunch:

**Comment by weerd** [comment]
:dancers: LibSass! :+1:

File: scss/_utilities/_variables.scss (hunk 1 above)
Comment (weerd) [inline scss/_utilities/_variables.scss]: haha

**Pull request merged**